packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -354,6 +354,12 @@ abstract contract MajorityVotingBase is
         actions = proposal_.actions;
     }
 
+    /// @notice Returns the current voting settings.
+    /// @return The current voting settings.
+    function getVotingSettings() public view virtual returns (VotingSettings memory) {
```

Comment:
I had to add this function because I considered implementing `isMember` in `TokenVoting` as
```solidity
function isMember(address _account) external view virtual returns (bool) {
    uint256 tokenBalance = votingToken.getVotes(_account);
    if (tokenBalance == 0) {
        return false;
    }
    if (tokenBalance < getVotingSettings().minProposerVotingPower) {
        return false;
    }
    return true;
}
```

However, since we decided for
```solidity
function isMember(address _account) external view virtual returns (bool) {
    return votingToken.getVotes(_account) > 0;
}
```
`getVotingSettings().minProposerVotingPower)` is actually not needed. 
I would say we can delete it.

The `VotingSettings private votingSettings;` are private, but the UI can get the current settings from the `VotingSettingsUpdated` events that we emit if they are changed.

Reply:
Don't forget to delete it. Currently, it still contains the function.

Reply:
As a point of view, what if inheritting contract needs voting settings to do something ? currently, without VotingSettings function, it would need to call separate functions to get all values. which is kind of not ideal. It seems returning VotingSettings function seem better, but then it would be hard to allow overriding specific functions(e.x. `minProposerVotingPower`, `votingMode`)..

On the other hand, having both(function and separate functions) seem redundant. happy to quickly hear your opinion.

Reply:
I wouldn't plan with more people using it. If we don't need it now, I would remove it. We can later still upgrade the plugins. What do you think?

Reply:
Works for me..

Reply:
Removed in 02094593bced31235717499b0d0c91872f9720b4